ext/drmgr/drmgr.h

```diff
@@ -721,6 +721,82 @@ DR_EXPORT
 ptr_uint_t
 drmgr_reserve_note_range(size_t size);
 
+/***************************************************************************
+ * EMULATION
+ */
+
+/**
+ * Holds data about an emulated instruction, typically populated by an emulation
+ * client and read by an observational client.
+ */
+typedef struct _emulated_instr_t {
+    app_pc pc;            /**< The PC address of the emulated instruction. */
+    instr_t *instr;       /**< The emulated instruction. */
+    unsigned int version; /**< The version of the ISA the instruction belongs to. */
```

Comment:
I am probably missing something, but I am not sure why we need that field here? Shouldn't the ISA version be obtainable from the instr_t? (I know there's no API for that currently, but wouldn't instr_t be a better place to get that info?)

Reply:
Good point. `instr_t` seems to be the logical place to store the ISA version an instruction belongs to. I've tried to make this PR as un-intrusive as possible for now hence its current location. I'll leave it up to Derek to decide what is the best approach.

Reply:
Xref #431
If we have the ISA version here and the 'emulation' client has to set it, the 'version' becomes a agreement between emulation and analysis client, rather than being provided by DynamoRIO itself, coupling them more than necessary. But it might be useful to have this flexibility if the clients emulate instructions DynamoRIO core does not know about.

Reply:
I've removed the version field. This is not a current requirement for emulation functionality. The method by which the version of instructions (emulated or otherwise) is established requires more thought in the broader DynamoRIO context.